data/tests/in1dataset.xml

```diff
@@ -1,2 +1,2 @@
 <?xml version='1.0' encoding='us-ascii'?>
-<dataset><datasetName>datasetNameTest</datasetName><criteria>11</criteria><categories>4</categories><alternatives>7</alternatives><alternative>x1<g1>2</g1><g2>3</g2><g3>3</g3><g4>2</g4><g5>3</g5><g6>4</g6><g7>2</g7><g8>1</g8><g9>2</g9><g10>3</g10><g11>5</g11><assignment>3</assignment></alternative><alternative>x2<g1>1</g1><g2>1</g2><g3>1</g3><g4>1</g4><g5>1</g5><g6>1</g6><g7>1</g7><g8>4</g8><g9>2</g9><g10>3</g10><g11>5</g11><assignment>2</assignment></alternative><alternative>x3<g1>1</g1><g2>1</g2><g3>1</g3><g4>1</g4><g5>1</g5><g6>1</g6><g7>1</g7><g8>1</g8><g9>1</g9><g10>1</g10><g11>1</g11><assignment>1</assignment></alternative><alternative>x4<g1>3</g1><g2>3</g2><g3>3</g3><g4>2</g4><g5>4</g5><g6>3</g6><g7>3</g7><g8>5</g8><g9>1</g9><g10>3</g10><g11>5</g11><assignment>4</assignment></alternative><alternative>x5<g1>1</g1><g2>1</g2><g3>1</g3><g4>1</g4><g5>1</g5><g6>1</g6><g7>1</g7><g8>1</g8><g9>1</g9><g10>1</g10><g11>1</g11><assignment>1</assignment></alternative><alternative>x6<g1>1</g1><g2>3</g2><g3>2</g3><g4>2</g4><g5>1</g5><g6>3</g6><g7>2</g7><g8>1</g8><g9>1</g9><g10>3</g10><g11>5</g11><assignment>2</assignment></alternative><alternative>x7<g1>1</g1><g2>3</g2><g3>2</g3><g4>1</g4><g5>1</g5><g6>1</g6><g7>2</g7><g8>1</g8><g9>3</g9><g10>3</g10><g11>1</g11><assignment>2</assignment></alternative></dataset>
```

Comment:
Is it normal that you didn't downgrade all categories by one?

Reply:
yes totally 
that is the purpose of the pr

Reply:
I really don't understand what you did here, but if you are sure of it you can merge

Reply:
It parts of the thing I spoke about yesterday. 
My algo stipulates that cat ranks start a 0 not 1 as in the dataset we were using